_episodes/09-conclusion.md

```diff
@@ -57,23 +57,36 @@ The following figure shows the dependencies involved in building the `all` targe
 
 > ## Creating an Archive
 >
-> Add new rules, update existing rules, and add new macros to:
+> Often it is useful to create an archive file of your project that includes all data, code, 
+> and results. An archive file can packages many files into a single file that can easily be shared
+> with collaborators or downloaded as a single file. We can add steps to create the archive 
+> file inside the Makefile itself so it's easy to update our archive file as the project changes. 
+>
+> 
+> Edit the Makefile to create an archive file of your project.  Add new rules, update existing
+> rules, and add new macros to:
```

Comment:
Agreed, "macro" shows up in the PNG exercise too and I was confused by that.

Reply:
It's introduced in episode 6 as an equivalent term for variable used in some versions of Make. However, I've just updated episode 9 to use variable to avoid confusion.